src/mmrelay/meshtastic_utils.py

```
@@ -435,6 +568,305 @@ def _get_device_metadata(client: Any) -> dict[str, Any]:
+                client = BleakClient(address)
+
+                is_connected = None
+                is_connected_method = getattr(client, "is_connected", None)
+
+                if is_connected_method and callable(is_connected_method):
+                    is_connected = await is_connected_method()
+                elif isinstance(is_connected_method, bool):
+                    is_connected = is_connected_method
+                else:
+                    is_connected = False
+            except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)